### PR TITLE
adds couch2pg tutorial

### LINKED ISSUE #625: Add "local setup" style guide for couch2pg
There's been some discussion recently in the forums (exhibit [A](https://forum.communityhealthtoolkit.org/t/replicating-data-using-couch2pg-to-postgres-local-db/1696/6) and [B](https://forum.communityhealthtoolkit.org/t/a-more-complete-docker-compose-file/1658)) where developers got stuck on how to set up `couch2pg`. I think if we did a version of the ["local setup" tutorial](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/), but for `couch2pg` this would be really helpful.  We could, and should, leverage the local CHT instance set up locally on this existing page.

By publishing easier to follow instructions for getting a development instance up and running, app devs can better understand the data, metedata and telemetry that a RDBMs filled with CHT data offers.

### PR DESCRIPTION
**Pull request by kitsao**
Closes #625

### CODE REVIEW

**Review by eljhkrr — COMMENTED** [review]
This is great. I've added some info from the troubleshooting we did some time ago.

File: content/en/apps/tutorials/couch2pg-setup.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+To read more about environmental variables, see the [CHT Couch2pg readme](https://github.com/medic/cht-couch2pg#readme).
+
+## Required Resources
+
+Before you begin, you need to have some useful software and tools that are required for things to work:
+
+* [nodejs](https://nodejs.org/en/) 8 or later
+* [npm](https://www.npmjs.com/get-npm)
+* [PostgreSQL](https://www.postgresql.org/) 9.4 or later
```
[the remark this answers is not in the document]
Comment (kitsao) [reply content/en/apps/tutorials/couch2pg-setup.md:44]: I think this issue was handled in [cht-couch2pg #117](https://github.com/medic/cht-couch2pg/pull/117), but I've captured it under known issues.

**Comment by garethbowen** [comment]
@eljhkrr It looks like your feedback has been incorporated - can you please have another look at reviewing this PR when you get a chance?

**Review by eljhkrr — APPROVED** [review]
LGTM

**Pull request merged**